src/main/java/dev/galacticraft/mod/content/block/machine/SimpleMultiBlockMachineBlock.java

```diff
@@ -51,6 +56,21 @@ protected SimpleMultiBlockMachineBlock(Properties properties, ResourceLocation f
         this.partState = partBlock.defaultBlockState();
     }
 
+    @Override
+    protected void onExplosionHit(BlockState state, Level world, BlockPos pos, Explosion explosion, BiConsumer<ItemStack, BlockPos> stackMerger) {
+        if (state.getBlock() != GCBlocks.ADVANCED_SOLAR_PANEL && state.getBlock() != GCBlocks.BASIC_SOLAR_PANEL) {
+            return;
+        }
```

Comment:
It can be assumed that the block state is a multiblock (these checks aren't necessary)